### PR TITLE
Branch audit

### PR DESCRIPTION
**Pull request by bragur**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/audit/branch-audit**](https://docs-eaprrd28i.avo-dev.app/audit/branch-audit) | [**/audit/overview**](https://docs-eaprrd28i.avo-dev.app/audit/overview) |
|:-:|:-:|
|<a href="https://docs-eaprrd28i.avo-dev.app/audit/branch-audit">Screenshot of /audit/branch-audit</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/m0nr4kgkDWPg-AVYcDMAwQpZgia4oCpzaINYFclkUA76B-8dlTUTgs3r5-ApJQS7xlBAdNcGtdWNJRf5Zl6xyA.png">(view full size)</a> |<a href="https://docs-eaprrd28i.avo-dev.app/audit/overview">Screenshot of /audit/overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ANB1WzBoRVY2RS3C5jAwVn1utRUHaSjkZwN6RoBpOk7IwS8wE-kXogFROFsJsTsEDDDDJrAwfG3o5RZhDMN5tA.png">(view full size)</a> |







Commit 98b246b3dfadb2ad75faa85e65e083d5c9df6aaa (https://docs-eaprrd28i.avo-dev.app).

File: pages/audit/branch-audit.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+layout: 'guide'
+title: 'Branch Audits'
+#abstract: 'Learn how to use branch audits to identify and resolve issues introduced on branches.'
+---
+
+import Link from '../../components/Link';
+
+> **Note:** Branch Audit is a feature for the _Team_ and _Enterprise_ plans
+
+The branch audit review the changes you have made on your branch and flags issues which are introduced by your changes, such as unexpected naming or inconsistent types — helping you keep your tracking plan clean and consistent.
```
Comment (bjornj12) [inline pages/audit/branch-audit.mdx:11]: reviews*

File: pages/audit/branch-audit.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  <img
+    src={require('../../images/audit/auto-correct.png')}
+    width="75%"
+    alt="Example of multiple issues being fixed at once when there is inconsistancy in the casing of names of items."
+  />
+</center>
+<br />
+<br />
+
+For Enterprise plans you have the possibility of enforcing that there are no issues introduced on branches before they are merged.
```
Comment (bjornj12) [inline pages/audit/branch-audit.mdx:49]: I feel like we could set like a h2 here with 
"Enforcement of 0 issue policy" or something like that

File: pages/audit/overview.mdx [hunk 3]
```diff
@@ -10,6 +10,8 @@ The Avo tracking plan audit reviews your event and property definitions and flag
 
 The tracking plan audit monitors every change you make in your Avo Tracking Plan and highlights if new issues arise. You can think about the tracking plan audit like a spell-checker for your tracking plan.
 
+Furthermore, all changes on branches are also checked in the <Link passHref href="/audit/branch-audit"><a>Branch Audit</a></Link> to see if you have introduced any new issues on your branch.
```
Comment (bjornj12) [inline pages/audit/overview.mdx:13]: Unrelated to your changes specifically, but we are really missing a picture here of the tracking plan audit in the sidebar so people know what we are talking about 🤔

**Review by bjornj12 — APPROVED** [review]

**Review by hugihlynsson — APPROVED** [review]

**Pull request merged**